ueventd.rc

```diff
@@ -13,10 +13,7 @@
 /sys/bus/iio/devices/iio:device* scan_elements/in_anglvel_y_en 0660 system system
 /sys/bus/iio/devices/iio:device* scan_elements/in_anglvel_z_en 0660 system system
 /sys/bus/iio/devices/trigger* 	 name 0660 system system
-/sys/class/backlight*		 brightness 0664 system system
-/sys/class/backlight*		 max_brightness 0664 system system
```

Comment:
It's preferred to set file permissions in ueventd.rc. If the max_brightness line isn't needed, can't we just delete that one line?

My intention is to move all of these to system/core